tests/Rule/ContainsLowercaseCharactersTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Tests\Morebec\Validator\Rule;
+use InvalidArgumentException;
+use Morebec\Validator\Rule\ContainsLowercaseCharacters;
+use PHPUnit\Framework\TestCase;
+
+
+class ContainsLowercaseCharactersTest extends TestCase
+{
+    public function testValidate(){
+        $ruleFirst = new ContainsLowercaseCharacters(1,true);
+        $ruleSecond = new ContainsLowercaseCharacters(1,false);
+        $ruleThird= new ContainsLowercaseCharacters(1,false,"Custom Message");
```

Comment:
It would be better to test using different number of character values.
The value 1 here causes some false positives.